packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/usetoken.md

```diff
@@ -1,26 +1,24 @@
-To make authenticated requests to your resource server, get the access token from the `SessionClient`. The function below sends a request using `HttpURLConnection`:
+To make authenticated requests to your resource server, add the access token interceptor to your `OkHttpClient` instance.
```

Comment:
This is partially repeating the general (see below). The focus of this content is the how. You don't need to say "to make authorized requests... etc. It should be how to make those requests (the iOS version is fairly simple as there's a convenience function that does everything for you.)

I think this may be code that creates that function for the developer. If so, then the text is something like:
Add the following code to <file>. It adds a convenience function that... etc.


> ## Use the access token
> 
> Your own server API (a resource server in OAuth 2.0) uses the Okta-generated access token for both authenticating that the user is signed in and ensuring that the user is authorized to perform the action or access the information.
> 
> Use the access token by adding it to the HTTP `Authorization` header of outgoing requests in your mobile or other client using this format:
> 
> ```
> Authorization: Bearer ${token}
> ```
> 
> Your API then checks incoming requests for valid tokens. To learn how to protect your API endpoints and require token authentication, see [Protect your API endpoints](/docs/guides/protect-your-api/).
> 
> <StackSnippet snippet="usetoken" />
>